js/actions/appActions.js

```diff
@@ -255,12 +255,14 @@ const appActions = {
    * A request for a URL load
    * @param {number} tabId - the tab ID to load the URL inside of
    * @param {string} url - The url to load
+   * @param {boolean} forceReload - would you like to force reload provided tab
    */
-  loadURLRequested: function (tabId, url) {
+  loadURLRequested: function (tabId, url, forceReload) {
```

Comment:
can we make it clearer that this will only reload if the url is the same, perhaps naming the param something like `reloadMatchingUrl` or `reloadIfMatching`?

Reply:
updated